tempesta_fw/sock.c

```diff
@@ -113,14 +114,25 @@ ss_skb_entail(struct sock *sk, struct sk_buff *skb)
 	tcb->seq     = tcb->end_seq = tp->write_seq;
 	tcb->tcp_flags = TCPHDR_ACK;
 	tcb->sacked  = 0;
```

Comment:
Do we need tabs here before the `=` character? An attempt to beautify this code has failed anyway. :) Just add `=` after a space, and that's it.